src/index.html

```diff
@@ -7,12 +7,67 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="use-credentials"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="/src/styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__img"
+        src="/src/images/imac.jpeg"
+        alt="iMac"
```

Comment:
alt attribute should describe the image content
<img width="468" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/63243136/81bb1e60-9c46-4630-b66c-027ab6cee7b9">